### PR TITLE
Add FAQs about signing keys and YouTube API

### PR DESCRIPTION
**Pull request by TobiGr**
The part about the YouTube API is written in favour of https://github.com/TeamNewPipe/NewPipe/issues/1990.
I tried to keep the FAQ about the signing key problem as short and simple as possible. If you think we should add more info, we can do this.

CC @theScrabi @TheAssassin @comradekingu

### CODE REVIEW

File: _faq/download-signing-keys.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+categories:
+- download
+type: info
+title: "Upgrading NewPipe fails"
+---
+<section>
+    <p>This might happen because you installed the F-Droid version and want to update with an APK from GitHub or vice versa. Android blocks an app upgrade when the signing keys of the installed and the upgrading APK differ. The idea behind this is to protect users from apps which do not come from the original developer and therefore might be malware and a security risk for the user.</p>
+    <p>This means, you need to either get the APK from the correct source or backup your NewPipe database, uninstall and reinstall the app.</p>
```
[the remark this answers is not in the document]
Comment (TobiGr) [reply _faq/download-signing-keys.html]: Should be done now. However, I did not find a good solution to link the tutorial. Unfortunaltely, I couldn't get the permalink / path which is set in the config.yml.

File: _faq/download-signing-keys.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+categories:
+- download
+type: info
+title: "Upgrading NewPipe fails"
+---
+<section>
+    <p>This might happen because you installed the F-Droid version and want to update with an APK from GitHub or vice versa. Android blocks an app upgrade when the signing keys of the installed and the upgrading APK differ. The idea behind this is to protect users from apps which do not come from the original developer and therefore might be malware and a security risk for the user.</p>
```
Comment (comradekingu) [inline _faq/download-signing-keys.html]:
```suggestion
    <p>Maybe you installed the F-Droid version and want to update with an APK from GitHub or vice versa? Android does not allow app upgrades when the signing keys differ. This ensures the original developer has condoned the app.</p>
```

File: _faq/download-signing-keys.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    <p>This means, you need to either get the APK from the correct source or backup your NewPipe database, uninstall and reinstall the app. A <b class="text-danger">backup is necessary</b>, otherwise you will <b>loose your current app data</b> including your subscriptions, history, playlists and settings. If you need help with this, please follow <a href="tutorials/import-export-subscriptions/#">this tutorial</a>.</p>
+    <p>The overall solution to this problem is to configure and set up reproducible builds for NewPipe in F-Droid. We are working on this together with F-Droid developers and you can follow its progress <a href="https://github.com/TeamNewPipe/NewPipe/issues/1981">here</a>.</p>
```
Comment (comradekingu) [inline _faq/download-signing-keys.html]:
```suggestion
    <p>The final solution to this problem is to have reproducible builds for NewPipe in F-Droid. We are working on this with the F-Droid developers <a href="https://github.com/TeamNewPipe/NewPipe/issues/1981">here</a>.</p>
```

File: _faq/download-signing-keys.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+categories:
+- download
+type: info
+title: "Upgrading NewPipe fails"
+---
+<section>
+    <p>This might happen because you installed the F-Droid version and want to update with an APK from GitHub or vice versa. Android blocks an app upgrade when the signing keys of the installed and the upgrading APK differ. The idea behind this is to protect users from apps which do not come from the original developer and therefore might be malware and a security risk for the user.</p>
+    <p>This means, you need to either get the APK from the correct source or backup your NewPipe database, uninstall and reinstall the app. A <b class="text-danger">backup is necessary</b>, otherwise you will <b>loose your current app data</b> including your subscriptions, history, playlists and settings. If you need help with this, please follow <a href="tutorials/import-export-subscriptions/#">this tutorial</a>.</p>
```
Comment (comradekingu) [inline _faq/download-signing-keys.html]:
```suggestion
    <p>This means, you need to either get the APK from the same source, or backup your NewPipe database, uninstall and reinstall the new app. Without a backup, <b>current app data is lost</b>, including subscriptions, history, playlists and settings. Follow <a href="tutorials/import-export-subscriptions/#">this tutorial</a> to do so.</p>
```

**Comment by TobiGr** [comment]
@comradekingu Thanks again!

**Pull request merged**